src/components/Modal/README.md

```diff
@@ -42,8 +42,11 @@ Modals are overlays that prevent merchants from interacting with the rest of the
 
 Passing an API key to the [app provider component](https://polaris.shopify.com/components/structure/app-provider#section-initializing-the-shopify-app-bridge) causes the modal component to delegate to the [Shopify App Bridge](https://help.shopify.com/en/api/embedded-apps/app-bridge) instead of rendering as it would in a stand-alone application.
 
-Note in the props table that a number of properties are only available in stand-alone applications, and won't work in an embedded context.
+Note in in an embedded application context all documented properties are not be available. Some properties are only available in stand-alone applications.
 
+Properties which are available only in a stand-alone context are documented as `(stand-alone app use only)`. For instance the `children` property is documented as `(stand-alone app use only)`.
```

Comment:
I was going to create an explicit list of properties which are not available in an embedded app however this list could go out of date easily. Instead I picked a property which is most likely to be used and hardest to implement.